code-reviews/README.md

```diff
@@ -0,0 +1,44 @@
+# Code reviews
+## General
+- For discussions that are happening outside a PR, summarize them afterwards as
+  a comment for future reference.
+- Code reviews are discussions where everybody is encouraged to chime in
+  irrespective of their job title.
+- Words such as "just", "simply", and "easily" tend to pass judgment about the
+  difficulty of a problem.
+
+## Giving feedback
+- Always read your comment from the perspective of the author of the pull
+  request.
+- If you see something good, say something good.
+- Suggest specific changes rather than using phrases such as "What do you think
+  about clarifying X?" and explain why a change might be appropriate.
+- Avoid selective ownership of code ("yours", "mine" etc.).
+- Consider reading the code changes from the bottom to the top to catch issues
+  that others might overlook (as most people tend to read the code from the
+  top to bottom).
+
+## Receiving feedback
+- The feedback is not a critique of you but the code. 
+- Be grateful for the reviewer's suggestions.
+- Provide context to the pull request and explain the motivation for it.
+- Add comments to specific lines where you want feedback or have questions.
+- Add comments to specific lines where you anticipate questions and explain why
+  you changed something.
+- Every comment should be seen as a suggestion. Ask questions until you
+  understand it and agree that the change would be an improvement.
+- Tag the appropriate people in the PR (`git blame` can be useful here. Bonus
+  points for aliasing `git blame` to `git praise`).
```

Comment:
It already has the alias `git annotate`, so it could be rephrased to

```suggestion
- Tag the appropriate people in the PR (`git annotate` can be useful here).
```

But I do acknowledge that `git praise` is positive, where `git annotate` is neutral.

Reply:
`git annotate` is new to me but I agree that it makes sense to change the sentence. Thanks!